lib/config.coffee

```diff
@@ -68,4 +68,9 @@ module.exports = {
     default: ''
     description: 'Hash of the last backup restored or created'
     order: 14
+  quietUpdateCheck:
+    type: 'boolean'
+    default: false
+    description: 'Mute \'Latest backup is already applied\' message'
```

Comment:
Can you please use double quotes outside. Then the inside single quotes don't have to be escaped.